graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpConsumer.java

```diff
@@ -132,6 +136,15 @@ public void connect() throws IOException {
         factory.setPort(port);
         factory.setVirtualHost(virtualHost);
 
+        if (tls) {
+            try {
+                LOG.info("Enable TLS for AMQP input [{}/{}].", sourceInput.getName(), sourceInput.getId());
+                factory.useSslProtocol();
+            } catch (NoSuchAlgorithmException | KeyManagementException e) {
+                throw new IOException(e);
```

Comment:
Maybe add a simple message like "Couldn't enable TLS for AMQP input." to the exception.